druid-shell/src/hotkey.rs

```diff
@@ -109,7 +109,7 @@ impl HotKey {
 
     /// Returns `true` if this [`KeyboardEvent`] matches this `HotKey`.
     ///
-    /// [`KeyboardEvent`]: keyboard_types::KeyEvent
+    /// [`KeyboardEvent`]: KeyEvent
```

Comment:
This fixes the link, but the text should read "KeyEvent" as well. Good catch though!